src/sass/components/govcy-header.scss

```diff
@@ -88,37 +100,39 @@ $govcy-sub-menu-color:#577589;
 		padding: 0.1rem;
 		display:flex;
 		align-items: center;
-    color: white;
+		flex-direction: row;
```

Comment:
This mixes tabs and spaces, again whitespace issue, not really a problem but we should agree tabs vs spaces

Reply:
on my code i don't have a space issue